### PR TITLE
updated dependencies for QuanTP

### PR DESCRIPTION
**Pull request by jraysajulga**
Hello. Just trying to fix some dependency issues.

### CODE REVIEW

File: tools/quantp/quantp.xml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+        <requirement type="package" version="4.8.0">r-plotly</requirement>
+        <requirement type="package" version="0.6.1.1">r-d3heatmap</requirement>
         <requirement type="package" version="3.0.1">r-gplots</requirement>
         <requirement type="package" version="0.7.6">r-dplyr</requirement>
         <requirement type="package" version="3.0.0">r-ggplot2</requirement>
         <requirement type="package" version="0.4.5">r-ggfortify</requirement>
-        <requirement type="package" version="4.8.0">r-plotly</requirement>
-        <requirement type="package" version="0.6.1.1">r-d3heatmap</requirement>
-        <requirement type="package" version="0.5.0">r-munsell</requirement>
+        <requirement type="package" version="1.6.36">libpng</requirement>
```
Comment (bgruening) [inline tools/quantp/quantp.xml]: libpng should not be needed...

Comment (jraysajulga) [reply tools/quantp/quantp.xml]: Okay. Need to do more testing.

**Comment by jraysajulga** [comment]
Okay. Thanks. Looks like it passed? @bgruening

**Comment by bgruening** [comment]
But it worked also before, isn't it? It was merged :) So not sure what the issue was :)

**Comment by jraysajulga** [comment]
Been running into dependency issues on [z.umn.edu/proteogenomicsgateway](http://galaxyp-proteogenomics.duckdns.org). It's throwing R script errors `Fatal error: Exit code 127 ()
/export/galaxy-central/database/job_working_directory/016/16172/tool_script.sh: line 9: Rscript: command not found` with the addition of r-plotly and r-d3heatmap.

**Comment by bgruening** [comment]
That looks more like a installation problem then a dependency problem. I guess re-creating the env should be enough.

**Comment by jraysajulga** [comment]
Ok. We got it working. Thanks for taking a look at this.

**Pull request closed without merging**